core/src/rpc_input.rs

```diff
@@ -0,0 +1,22 @@
+use solana_sdk::commitment_config::CommitmentConfig;
+
```

Comment:
Shouldn't the contents of this file live in `client/src/rpc_request.rs` for use by `client/src/rpc_client.rs`?

Reply:
For future use, you mean? Or should we plumb `getLargestAccounts into client/cli at this time?

Reply:
Either or, whatever avoids adding the file `core/src/rpc_input.rs` so I don't feel compelled to bikeshed the name :)

 A presence in `client/src/rpc_request.rs` for each new RPC API we add, at the time the new API is added, would be my preference in general.